.codeclimate.yml

```diff
@@ -37,3 +37,9 @@ plugins:
         enabled: false
   shellcheck:
     enabled: true
+exclude_patterns:
+  - "**/*_generated.go"
+  # default excludes are overwritten, add them again
+  # https://docs.codeclimate.com/docs/excluding-files-and-folders#section-auto-generated-file-and-folder-exclusions
```

Comment:
The default list contains about 20 entries for various languages, I added only the necessary one's.